README.md

```diff
@@ -0,0 +1,127 @@
+|[`scientific.notation()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/scientific.notation.html)|Use scientific notation in plots|
+|[`show.available.palettes()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/show.available.palettes.html)|Display the available colour palettes|
+|[`thousands.split()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/thousands.split.html)|Divide strings into groups of thousands|
+|[`write.metadata()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/write.metadata.html)|Writes Metadata|
+|[`write.plot()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/write.plot.html)|Simplifies plotting by standardizing and centralizing all output-handling|
+
+## Resources
+Available resources on BPG usage can be found at the package [CRAN page](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/index.html), [reference manual](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/BoutrosLab.plotting.general.pdf), or [vignette](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/vignettes/PlottingGuide.pdf).
+
+## Getting help
+Looking for guidance or support with Boutros.plotting.general? Check out our [Discussions](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/discussions) page.
+
+Submit bugs, suggest new features or see current works at our [Issues](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/issues) page.
+
+[Pull requests](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/pulls) are also open for discussion.
+
+## Contributors
+For lists of contributors please visit [here](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/authors.html) and [here](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/graphs/contributors) at GitHub
+
+## Citation information
+P’ng, C., Green, J., Chong, L.C. et al. BPG: Seamless, automated and interactive visualization of scientific data. *BMC Bioinformatics* **20**, 42 (2019). https://doi.org/10.1186/s12859-019-2610-2
+
+## License
+
+Authors: Christine P’ng, Jeffrey Green, Lauren C. Chong, Daryl Waggott, Stephenie D. Prokopec, Mehrdad Shamsi, Francis Nguyen, Denise Y. F. Mak, Felix Lam, Marco A. Albuquerque, Ying Wu, Esther H. Jung, Maud H. W. Starmans, Michelle A. Chan-Seng-Yue, Cindy Q. Yao, Bianca Liang, Emilie Lalonde, Syed Haider, Nicole A. Simone, Dorota Sendorek, Kenneth C. Chu, Nathalie C. Moon, Natalie S. Fox, Michal R. Grzadkowski, Nicholas J. Harding, Clement Fung, Amanda R. Murdoch, Kathleen E. Houlahan, Jianxin Wang, David R. Garcia, Richard de Borja, Ren X. Sun, Xihui Lin, Gregory M. Chen, Aileen Lu, Yu-Jia Shiah, Amin Zia, Ryan Kearns & Paul C. Boutros (PBoutros@mednet.ucla.edu)
```

Comment:
I think more recent authors should be added (the citation above is fine, but it's from ~2018 I think this should include everybody who's made a contribution to the R package since that time